boilerplate/index.js

```diff
@@ -8,22 +8,6 @@
 //
 // It's easier just to leave it here.
 import App from "./app/app.tsx"
-
-// Should we show storybook instead of our app?
-//
-// ⚠️ Leave this as `false` when checking into git.
-const SHOW_STORYBOOK = false
-
-let RootComponent = App
-if (__DEV__) {
-  if (SHOW_STORYBOOK) {
-    // Only include Storybook if we're in dev mode
-    const { StorybookUIRoot } = require("./storybook")
-    RootComponent = StorybookUIRoot
-  }
-}
-
 import { AppRegistry } from "react-native"
-AppRegistry.registerComponent("HelloWorld", () => RootComponent)
 
-export default RootComponent
+AppRegistry.registerComponent("HelloWorld", () => App)
```

Comment:
I like that this file got simpler. It should be really simple!

Reply:
I can add some verbiage to the demo screen today 👍

Reply:
Thanks, Frank, that'd be great - either add it to this PR, then merge it; or merge this, then add at your convenience.

Reply:
Let's get this in first to check it off and I'll make a separate PR or just push up